pymc3/variational/opvi.py

```diff
@@ -956,16 +955,21 @@ def __init_group__(self, group):
         self.input = self._input_type(self.__class__.__name__ + "_symbolic_input")
         # I do some staff that is not supported by standard __init__
         # so I have to to it by myself
-        self.group = [get_transformed(var) for var in self.group]
 
-        # XXX: This needs to be refactored
-        # self.ordering = ArrayOrdering([])
-        self.replacements = dict()
+        # 1) we need initial point (transformed space)
+        model_initial_point = self.model.initial_point
+        
+        # 2) we'll work with a single group, a subset of the model
+        # here we need to create a mapping to replace value_vars with slices from the approximation
+        start_idx = 0
         for var in self.group:
```

Comment:
@brandonwillard I have issues with my replacements, can you please review this part of the code in case I do it wrong.

Reply:
Do you need to replace variables in an existing graph, or create a `dict` that represents replacements that will be made elsewhere (e.g. populate something like the previous `self.replacements`)?